pkg/storageos/deploy_test.go

```diff
@@ -1527,31 +1527,37 @@ func TestDeployTLSEtcdCerts(t *testing.T) {
 }
 
 // TestDeployPodPriorityClass tests that the pod priority class is set properly
-// for the daemonset and statefulset pods when deployed in kube-system
-// namespace.
+// for the daemonset when deployed in kube-system namespace.
```

Comment:
Because the priority class is cluster scoped and the old behavior of kube-system exclusivity is removed, maybe this whole test can be removed. We don't set priority class on anything except the daemonset and the daemonset will always have the priority class set, regardless of the namespace.

Reply:
Since we still only set the DaemonSet priority class when it runs in `kube-system`, we should probably keep the test until we remove that restriction.  I can do that in a separate PR.  I think we still allow deploying CSI as a StatefulSet, so I kept those tests too, though I agree they're not very useful.